lib/workers/repository/update/pr/index.ts

```diff
@@ -27,6 +29,10 @@ import { ChangeLogError } from './changelog/types';
 import { prepareLabels } from './labels';
 import { addParticipants } from './participants';
 
+const prRenovateVersionsRe = regEx(
+  /<!--\s*renovate-pr-data\s*(?<json>{(?:\s*.*)+})\s*-->/
+);
```

Comment:
```suggestion
const prRenovateVersionsRe = regEx(
  /<!--\s*renovate-pr-data\s*(?<json>{.+?})\s*-->/
);
```
keep the regex simple, `.` includes space. not sure if non-greedy works here.

Reply:
it's not working with /s flag in code, just in the website, i'm not sure how to add it :/
`/<!--\s*renovate-pr-data\s*(?<json>{.+?})\s*-->/s`

Reply:
> but i dont think renovate will ever have "-->" in its versions, which is the only values we are adding

That's why I'm saying the error is *potential*. We can make it happen in the future with just two steps:
1. Introduce some field with the pretty much arbitrary content
2. Wait for it to contain `-->` at some moment

> i have no idea how base64 would help

Resulting regex will be more straightforward (something like `<!--renovate-pr-data:(?<payload>.*?)-->`), and we don't need to deal with spaces and newlines at all.